packages/eslint-plugin-ecmascript-compat/lib/rule.js

```diff
@@ -1,21 +1,36 @@
 const compatibility = require('./compatibility');
 const { createDelegatee, delegatingVisitor } = require('./delegation');
 const features = require('./features');
+const polyfilling = require('./polyfilling');
 const targetRuntimes = require('./targetRuntimes');
 
 const targets = targetRuntimes();
-const delegateeConfigs = compatibility
-  .forbiddenFeatures(features, targets)
-  .map((feature) => feature.ruleConfig);
+const forbiddenFeatures = compatibility.forbiddenFeatures(features, targets);
 
 module.exports = {
   meta: {
     type: 'problem',
-    schema: [], // no options
+    schema: [
+      {
+        "type": "object",
+        "properties": {
+          "polyfills": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        },
+        "additionalProperties": "false",
+      }
+    ]
   },
   create(context) {
```

Comment:
`context` is always given, so we don't need to use optional chaining when accessing `options` on it